packages/slice-machine/lib/builders/CustomTypeBuilder/TabZone/index.tsx

```diff
@@ -44,14 +51,12 @@ const TabZone: React.FC<TabZoneProps> = ({ tabId, fields, sliceZone }) => {
     replaceCustomTypeSharedSlice,
   } = useSliceMachineActions();
 
-  const { currentCustomType, poolOfFields } = useSelector(
-    (store: SliceMachineStoreType) => ({
-      currentCustomType: selectCurrentCustomType(store),
-      poolOfFields: selectCurrentPoolOfFields(store),
-    })
-  );
+  const { poolOfFields } = useSelector((store: SliceMachineStoreType) => ({
+    poolOfFields: selectCurrentPoolOfFields(store),
+  }));
 
-  if (!currentCustomType || !poolOfFields) {
+  // TODO: fix this
```

Comment:
I would like to leave it as is because it's not typed and I'm not sure what's expected here. It's just very probable that the condition is never true

Reply:
So for me we could remove the TODO, a TODO without an issue and without any real idea behing is not a todo. Or it could be a simple comment. But she should be careful with TODO and we should use then only when we want to do someting. no?